components/builder-worker/src/runner/mod.rs

```diff
@@ -176,14 +178,47 @@ impl Runner {
             error!("Unable to clone remote source repository, err={}", err);
             return self.fail(net::err(ErrCode::VCS_CLONE, "wk:run:4"));
         }
+
+        self.workspace
+            .job
+            .set_build_start(UTC::now().to_rfc3339());
+
+        // TODO: We don't actually update the state of the job to
```

Comment:
Ah, I was seeing exactly this issue in the scheduler logging and going to investigate why :)